src/converters/piral-blazor/src/converter.ts

```diff
@@ -1,5 +1,34 @@
-import { ForeignComponent, BaseComponentProps } from 'piral-core';
-import { initialize, activate, deactivate, attachEvents } from './internal';
+import type { ForeignComponent, BaseComponentProps } from 'piral-core';
+import { initialize, activate, deactivate, attachEvents } from './interop';
+
+const globalEventNames = [
+  'abort',
+  'blur',
+  'change',
+  'error',
+  'focus',
+  'load',
+  'loadend',
+  'loadstart',
+  'mouseenter',
+  'mouseleave',
+  'progress',
+  'reset',
+  'scroll',
+  'submit',
+  'unload',
+  'DOMNodeInsertedIntoDocument',
+  'DOMNodeRemovedFromDocument',
+  'click',
+  'dblclick',
+  'mousedown',
+  'mousemove',
+  'mouseup',
+];
+
+function dispatchToRoot(event: Event) {
```

Comment:
If we refactor this a bit then `globalEventNames` and `dispatchToRoot` could be in `interop.ts`. This would allow to put `blazor-root` in a (module-local) variable that can be reused.